assets/chat/css/style.scss

```diff
@@ -97,7 +99,7 @@ a {
 hr {
   border: 0;
   margin: 1em 0;
-  border-top: 1px solid darken(a.$color-surface-dark1, 50%);
+  border-top: 1px solid color.adjust(a.$color-surface-dark1, $lightness: -50%);
```

Comment:
Though it is effectively the same - in production this color is `#000`, but here is `hsl(0, 0%, -48.8235294118%)` due to how sass handles the conversion. Consider just making this black, or adding another variable darker than `surface-dark1` to avoid this weird behavior?

Reply:
I'm sure we'll rework colors in the future someday. I don't mind weirdness in the generated CSS so long as the deprecation warnings are gone.